src/max_flow/max_flow.c

```
@@ -39,15 +39,17 @@ PGDLLEXPORT Datum _pgr_maxflow(PG_FUNCTION_ARGS);
 PG_FUNCTION_INFO_V1(_pgr_maxflow);
```

Cppcheck (2.19.0)

[information] Include file

(missingIncludeSystem)